backend/heal_all_datapackages.py

```
@@ -0,0 +1,150 @@
+                ).delete(synchronize_session=False)
+                print(f"  - Deleted {deleted_rows} cache entries for checksums: {', '.join(checksums)}")
+                
+                # Reset is_setup on rooms using these checksums
+                rooms = session.query(TrackedRoom.id, TrackedRoom.game_checksums_json).all()
+                reset_room_ids = []
+                for r_id, game_checksums_json in rooms:
+                    try:
+                        r_checksums = json.loads(game_checksums_json or '{}').values()
+                        if any(c in checksums for c in r_checksums):
+                            reset_room_ids.append(r_id)
+                    except:
```

Ruff (0.15.15)

[error] Do not use bare `except`

(E722)